nbclassic/shim_notebook.py

```diff
@@ -82,5 +82,6 @@ def shim_notebook():
     from jupyter_server import utils
     sys.modules["notebook.utils"] = utils
 
-    base.handlers.IPythonHandler = base.handlers.JupyterHandler
+    from jupyter_server.base import handlers
+    base.handlers.IPythonHandler = handlers.JupyterHandler
```

Comment:
This works fine, but FYI can be rewritten as:

```
    from jupyter_server.base.handlers import JupyterHandler as JHandler
    base.handlers.IPythonHandler = JHandler
```

Reply:
Thx. Both ways are fine I guess. I will merge as it.